Frontend/ui-common/src/Components/Banner/Controller.purs

```diff
@@ -49,7 +50,11 @@ type Config = {
   titleTextVisibility :: Boolean,
   imagePadding :: Padding,
   cornerRadius :: Number,
-  imageMargin :: Margin
+  imageMargin :: Margin,
+  actionTextBackgroundColor :: Maybe String,
```

Comment:
Can we put image and actionText related items in their corresponding combined configs